data/hai/hai missions.txt

```diff
@@ -1658,3 +1658,123 @@ mission "Strider notification"
+			choice
+				`	"Why would you want to go to <planet>? Isn't that an abandoned station?"`
+				`	"Sure, I can take you there."`
+					goto accept
+				`	"I'm not going that way. You'll need to find some other captain."`
+					decline
+			`	"Ah, you must not know about <planet>. There are many tales which swirl around the station, chief among them being the existence of ghosts and monsters haunting its corridors. We'd like to take advantage of the popular image of Darkrest in the creation of our horror film by using it as a backdrop. It's sort of the human equivalent of setting your film in a 'haunted' forest. Don't worry, we'll bring enough food to last us for the month."`
+			choice
+				`	"Why do people think it's haunted?"`
+				`	"Alright then, I can take you there."`
+					goto accept
+				`	"I'm not going that way. You'll need to find some other captain."`
+					decline
+			`	"Back when the station was still in use, there was a mysterious malfunction in one of the hatches which released oxygen out the airlock. Hundreds on board suffocated to death. To this day, no one knows what caused it, and legends have grown out of the event."`
+			choice
+				`	"What makes you think I'm parking my ship at a haunted station?"`
+				`	"Alright then, I can take you there."`
+					goto accept
+				`	"I'm not going that way. You'll need to find some other captain."`
+					decline
+				`	Likeet gives a laugh. "It's not actually haunted, of course. Those are just stories. Do you think I'd send an entire film crew and all their expensive equipment to a place that was actually dangerous?"`
+			choice
+				`	"Alright then, I can take you there."`
+				`	"I'm not going that way. You'll need to find some other captain."`
+					decline
+			label accept
+				`	"Excellent! I'll let the crew know where to find you."`
+					accept
+	on visit
+		dialog phrase "generic cargo and passenger on visit"
+	on complete
+		event "hai film crew" 30
+		payment 100000 50
+		conversation
+			`You help your passengers unload all of their film equipment once you arrive on <planet>. The props seem to get more and more elaborate as they are unpacked from your ship. You note the presence of countless rubber limbs, mechanical monsters, and cans of fake blood being carried onto the set. Once you are finished, Likeet approaches you.`
+			choice
+				`	"What's the name of your movie, anyways?"`
+				`	"What's your movie about, anyways?"`
+				`	"I better get going, then."`
+					goto going
+			`	"I don't want to spoil anything, but it's a psychological horror film that explores the nature of human isolation, and the shadow that our own mortality casts over our lives." She continues to elaborate on more philosophical themes that seemingly have less and less to do with the actual film, until she notices that your eyes have long since glazed over.`
+			`	"Well, anyways, thanks for your help, Captain. I'll contact you in a month once we're finished," says Likeet.`
```

Comment:
```suggestion
			`	"Well, anyways, thanks for your help, Captain. I'll contact you in a month once we're finished filming," says Likeet.`
```

This change probably isn't necessary (context makes it obvious), but as it currently is it implies that they're going to finish the entire movie, including editing, on Darkrest.